### PR TITLE
Replace registrator with direct bridge to consul via mesos_consul

### PR DESCRIPTION
**Pull request by tayzlor**

### CODE REVIEW

**Comment by wallies** [comment]
if this works, will be awesome to get rid of registrator

**Comment by tayzlor** [comment]
it works of course :hankey: any other feedback/comments or lgtm?

**Comment by wallies** [comment]
Lgtm +1

File: roles/mesos_consul/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+
+- name: deploy mesos_consul service
+  become: yes
```
Comment (enxebre) [inline roles/mesos_consul/tasks/main.yml]: Do we need both become?

File: roles/mesos_consul/templates/mesos_consul.service.j2 [hunk 2]
```diff
@@ -0,0 +1,22 @@
+[Unit]
+Description=Mesos Consul
```
Comment (enxebre) [inline roles/mesos_consul/templates/mesos_consul.service.j2]: Thinking if we should run this through the weave proxy, probably does not matter

Comment (enxebre) [inline roles/mesos_consul/templates/mesos_consul.service.j2]: Agree we @wallies if it works lgtm 🐗🐗

**Pull request merged**